ci-operator/config/openshift/instaslice-operator/openshift-instaslice-operator-next.yaml

```diff
@@ -84,19 +84,21 @@ tests:
       CONTROL_PLANE_NODE_TYPE: a2-highgpu-1g
       CONTROL_PLANE_ZONES: '[us-central1-f]'
       EMULATED_MODE: disabled
```

Comment:
the master nodes can be regular instance, they don't need to be gpu. I think we can remove `CONTROL_PLANE_NODE_TYPE` and `CONTROL_PLANE_ZONES` and use the default.

Reply:
Let me try that out